### PR TITLE
Homework for lesson 07

### PR DESCRIPTION
**Pull request by LilianaLukash**
home work for lesson 07

### CODE REVIEW

File: home-work7.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+            case "list":
+                for key in phonebook.keys():
+                    print(key)
+
+            case "show":
+                if com[1] in phonebook:
+                    print(phonebook[com[1]])
+                else:
+                    shname = input("please enter the name ")
```
Comment (klymenok) [inline home-work7.py]: Here should be some error message but not one more input. Write here something like "user `com[1]` is not found"

File: home-work7.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                name = input("Please enter new name")
+                number = input("Please enter phone number")
+                phonebook[name] = number
+                print(phonebook)
+
+            case "delete":
+                if com[1] in phonebook:
+                    phonebook.pop(com[1])
+                else:
+                    dname = input("Which name you want to delete?")
```
Comment (klymenok) [inline home-work7.py]: add just an error message here, you don't need another input here

File: home-work7.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        print("wrong command")
+    else:
+        match com[0]:
+            case "stats":
+                print(f"There are {len(phonebook)} numbers in your phone book ")
+
+            case "add":
+                name = input("Please enter new name")
+                number = input("Please enter phone number")
+                phonebook[name] = number
```
Comment (klymenok) [inline home-work7.py]: The task was not to override records. So it is better to check if such record already exists before add this record

**Review by klymenok — CHANGES_REQUESTED** [review]

File: home-work7.py (hunk 1 above)
Comment (LilianaLukash) [reply home-work7.py]: was changed, please see in new commit

File: home-work7.py (hunk 2 above)
Comment (LilianaLukash) [reply home-work7.py]: was changed, please see in new commit

File: home-work7.py (hunk 3 above)
Comment (LilianaLukash) [reply home-work7.py]: was changed, please see in new commit

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**